core-bundle/src/Command/MigrateCommand.php

```diff
@@ -84,6 +92,16 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return 1;
     }
 
+    private function backup(): void
+    {
+        $config = $this->backupManager->createCreateConfig();
+        $this->io->info(sprintf(
+            'Creating a database dump to "%s" with the default options. Use --no-backup to disable this feature.',
+            $config->getBackup()->getFilepath()
+        ));
+        $this->backupManager->create($config);
```

Comment:
what if the backup fails here?

Reply:
Then it will throw an exception which the command shows accordingly. This is on purpose.